articles/libraries/lock-ios/v2/customization.md

```diff
@@ -155,6 +299,16 @@ Color used as the Lock background color.
 }
 ```
 
+### backgroundImage
+
+Image used as the Lock background
+
+```swift
+.withStyle {
+  $0.backgroundColor = LazyImage?
```

Comment:
What is the source for this, wondering if I documented it incorrectly somewhere? As this should be like the icons e.g.

```
LazyImage(name: "company_logo")
```